src/utils/formatValue.js

```diff
@@ -54,4 +54,12 @@ const formatValue = (formatter, value) => {
   }
 };
 
+export const toPercent = (decimal) => `${(decimal * 100).toFixed(0)}%`;
```

Comment:
Can we use the above formatValue function.
You would pass in formatValue('0.1~%', value)